R/forestplot.R

```diff
@@ -144,10 +145,13 @@ srv_g_forest_tte <- function(id,
                              plot_height,
                              plot_width) {
   with_reporter <- !missing(reporter) && inherits(reporter, "Reporter")
+  with_filter <- !missing(filter_panel_api) && inherits(filter_panel_api, "FilterPanelAPI")
```

Comment:
practically `with_filter` isn't needed as as module depends on `filter_panel_api` as `experimentSpecServer` needs it.

Reply:
so if module depends on filter_panel_api then we should have `stop_if_missing(filter_panel_api, "this module requires a filter panel see ....")`

Reply:
`with_filter` above was replaced with `checkmate::assert_class(filter_panel_api, "FilterPanelAPI` in all modules using `experimentSpecServer`.